bases/renku_data_services/crc_api/config.py

```diff
@@ -92,6 +92,11 @@ def __post_init__(self):
                 defaults = ServerOptionsDefaults.parse_obj(safe_load(f))
             self.default_resource_pool = generate_default_resource_pool(options, defaults)
 
+    @property
+    def repo(self):
+        """Used by alembic to find repo."""
```

Comment:
```suggestion
        """Used by alembic for database migrations to get the object that satisfies the DataRepository protocol."""
```

Also please add the type annotation for what is returned. I see that you have a `DataRepository` protocol. We should put that as the return type annotation. This change makes the change to the docstring unnecessary so you can keep the docstring short.

Reply:
I've addressed this in https://github.com/SwissDataScienceCenter/renku-data-services/pull/21/commits/83d4098936b03ca33e0a94ef03c02268f3eb90d0